pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -68,6 +68,26 @@ const (
 	EventExternalAnnotationAdded string = "ExternalAnnotationAdded"
 )
 
+var (
+	_ reconcile.Reconciler = &ReconcileMachineHealthCheck{}
+)
+
+// ReconcileMachineHealthCheck reconciles a MachineHealthCheck object
+type ReconcileMachineHealthCheck struct {
+	// This client, initialized using mgr.Client() above, is a split client
+	// that reads objects from the cache and writes to the apiserver
+	client    client.Client
+	scheme    *runtime.Scheme
+	namespace string
+	recorder  record.EventRecorder
+}
+
+type target struct {
+	Machine mapiv1.Machine
+	Node    *corev1.Node
+	MHC     mapiv1.MachineHealthCheck
+}
+
```

Comment:
This seems to have moved without changes, what happened there? Can we put it back to reduce the diff please?

Reply:
To keep the class more readable consts, vars and structs are aligned to the top instead of being scattered all over.
In order to reduce the diff I've placed all the non-functional changes into a single commit - so the functional changes can be reviewed without this static noise.

Reply:
I would move them to a whole other PR